tools/run_benchmarks.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+# Usage: tools/run_benchmarks.sh "deep-image-96-angular" "new-benchmark-server"
+
+set -e
+set -x
+
+SCRIPT_PATH="$( cd "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )"
```

Comment:
`&>/dev/null` is a shortcut for `>/dev/null 2>&1`

```suggestion
SCRIPT_PATH="$( cd "$(dirname "$0")" &>/dev/null ; pwd -P )"
```

Reply:
Done. Thanks!